dateparser/date.py

```diff
@@ -254,6 +252,40 @@ def _is_valid_date_obj(self, date_obj):
         return True
 
 
+class DateData:
+    """
+    Class that represents the parsed data with useful information.
+    It can be accessed like a dict object.
+    """
+
+    def __init__(self, date_obj=None, period=None, locale=None, is_relative=None):
+        self.date_obj = date_obj
+        self.period = period
+        self.locale = locale
+        self.is_relative = is_relative
+
+    def __getitem__(self, item):
+        if not hasattr(self, item):
+            raise KeyError(item)
+        return getattr(self, item)
+
+    def __setitem__(self, key, value):
+        if not hasattr(self, key):
+            raise KeyError(key)
+        setattr(self, key, value)
+
+    def __repr__(self):
+        return str(self.__dict__)
+
+    def __contains__(self, item):
+        if hasattr(self, item) and getattr(self, item):
+            return True
+        return False
```

Comment:
:lipstick: 

```suggestion
        return hasattr(self, item) and getattr(self, item)
```

Or, even shorter:

```suggestion
        return bool(getattr(self, item, False))
```